_oasis

```diff
@@ -6,6 +6,10 @@ Authors:      Xapi Project
 License:      LGPL-2.1 with OCaml linking exception
 Plugins:      DevFiles(0.4), META (0.4)
 BuildTools:   ocamlbuild
+OCamlVersion: >= 4.01
```

Comment:
I think it's a bit redundant anyway - I think I had to add it to try out the OMake support, but I've backed that out now.